peachjam/models/judgment.py

```diff
@@ -76,6 +84,14 @@ def generate_serial_number(self):
         if self.pk:
             query = query.exclude(pk=self.pk)
 
+        serial_nums = [s["serial_number"] for s in query.values("serial_number")]
+
+        if self.serial_number_override:
+            if int(self.serial_number_override) in serial_nums:
+                raise ValidationError("Serial number already in use.")
+            else:
+                return self.serial_number_override
```

Comment:
@longhotsummer Do we not need to enforce uniqueness for mnc serial numbers?

Reply:
I've been thinking about this. In the short term, not having this support is preventing uploaders from adding new content for courts that assign MNCs themselves, where there is basically no chance of overlap. So I'd like to unblock that ASAP.

When I worked on this yesterday I expected the FRBR URI uniqueness to enforce unique serial numbers. However, I realised that the code actually allows duplicate FRBR URIs, because it thinks the two documents belong to the same work.

Can we merge this in to unblock and then re-evaluate?